__tests__/app.test.js

```diff
@@ -167,4 +167,42 @@ describe("/api/articles/:article_id/comments", () => {
             expect(body.msg).toBe("bad request!")
         })
     })
+})
+describe("PATCH /api/articles/:article_id", () => {
+    test(":) PATCH 201 - updates the article given article id, returns updated article", () => {
+        return request(app).patch("/api/articles/1").send({inc_votes: -5}).expect(201).then(({body}) => {
+            expect(body.updatedArticle).toMatchObject({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: expect.any(String),
+                votes: 95,
+            })
+        })
+    })
+    test(":) PATCH 201 - updates the article given body that has more properties than needed", () => {
+        return request(app).patch("/api/articles/1").send({inc_votes: 4, article_id: 5}).expect(201).then((({body}) => {
+            expect(body.updatedArticle).toMatchObject({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: expect.any(String),
+                votes: 104,
+            })
+        }))
+    })
+    test(":( PATCH 400 - returns bad request given malformed body or missing field requirement", () => {
+        return request(app).patch("/api/articles/1").send({author: "fds"}).expect(400).then(({body}) => {
+            expect(body.msg).toBe("bad request!")
+        })
+    })
+    test(":( PATCH 404 - returns article not found given invalid article id", () => {
+        return request(app).patch("/api/articles/434").send({inc_votes: 3}).expect(404).then(({body}) => {
+            expect(body.msg).toBe("article not found!")
+        })
+    })
```

Comment:
Looks like we're missing some tests here. So far, all of the review IDs are valid, and what happens if the client has the correct key on the body, but the data type for the value isn't supported?